diff -r 48309b14abb9 dwm.c
--- a/dwm.c	Sun Apr 15 11:41:18 2012 +0200
+++ b/dwm.c	Tue Apr 24 17:14:17 2012 +0200
@@ -36,6 +36,7 @@
 #include <X11/Xlib.h>
 #include <X11/Xproto.h>
 #include <X11/Xutil.h>
+#include <X11/XKBlib.h>
 #ifdef XINERAMA
 #include <X11/extensions/Xinerama.h>
 #endif /* XINERAMA */
@@ -95,6 +96,7 @@
 	Client *snext;
 	Monitor *mon;
 	Window win;
+   unsigned char kbdgrp;
 };
 
 typedef struct {
@@ -854,6 +856,7 @@
 			selmon = c->mon;
 		if(c->isurgent)
 			clearurgent(c);
+       XkbLockGroup(dpy, XkbUseCoreKbd, c->kbdgrp);
 		detachstack(c);
 		attachstack(c);
 		grabbuttons(c, True);
@@ -1117,6 +1120,7 @@
 	Client *c, *t = NULL;
 	Window trans = None;
 	XWindowChanges wc;
+   XkbStateRec kbd_state;
 
 	if(!(c = calloc(1, sizeof(Client))))
 		die("fatal: could not malloc() %u bytes\n", sizeof(Client));
@@ -1171,6 +1175,8 @@
 	c->mon->sel = c;
 	arrange(c->mon);
 	XMapWindow(dpy, c->win);
+   XkbGetState(dpy, XkbUseCoreKbd, &kbd_state);
+   c->kbdgrp = kbd_state.group;
 	focus(NULL);
 }
 
@@ -1795,6 +1801,8 @@
 
 void
 unfocus(Client *c, Bool setfocus) {
+   XkbStateRec kbd_state;
+
 	if(!c)
 		return;
 	grabbuttons(c, False);
@@ -1803,6 +1811,8 @@
 		XSetInputFocus(dpy, root, RevertToPointerRoot, CurrentTime);
 		XDeleteProperty(dpy, root, netatom[NetActiveWindow]);
 	}
+ 	XkbGetState (dpy, XkbUseCoreKbd, &kbd_state);
+	c->kbdgrp = kbd_state.group;
 }
 
 void
