diff -up dwm-original/drw.c dwm/drw.c
--- dwm-original/drw.c	2015-09-29 14:21:19.564564127 +0200
+++ dwm/drw.c	2015-09-29 14:16:36.119603016 +0200
@@ -211,9 +211,9 @@ drw_rect(Drw *drw, int x, int y, unsigne
 	XSetForeground(drw->dpy, drw->gc, invert ? drw->scheme->bg->pix : drw->scheme->fg->pix);
 	dx = (drw->fonts[0]->ascent + drw->fonts[0]->descent + 2) / 4;
 	if(filled)
-		XFillRectangle(drw->dpy, drw->drawable, drw->gc, x+1, y+1, dx+1, dx+1);
+		XFillRectangle(drw->dpy, drw->drawable, drw->gc, x+6, y+4, dx+1, dx+1);
 	else if(empty)
-		XDrawRectangle(drw->dpy, drw->drawable, drw->gc, x+1, y+1, dx, dx);
+		XDrawRectangle(drw->dpy, drw->drawable, drw->gc, x+6, y+4, dx, dx);
 }
 
 int
diff -up dwm-original/dwm.c dwm/dwm.c
--- dwm-original/dwm.c	2015-09-29 14:21:19.564564127 +0200
+++ dwm/dwm.c	2015-09-29 14:13:45.418616769 +0200
@@ -1511,7 +1511,7 @@ setup(void) {
 	drw_load_fonts(drw, fonts, LENGTH(fonts));
 	if (!drw->fontcount)
 		die("No fonts could be loaded.\n");
-	bh = drw->fonts[0]->h + 2;
+	bh = drw->fonts[0]->h + 12;
 	updategeom();
 	/* init atoms */
 	wmatom[WMProtocols] = XInternAtom(dpy, "WM_PROTOCOLS", False);
Common subdirectories: dwm-original/.git and dwm/.git
