diff -up dmenu/dmenu.c dmenu-modified/dmenu.c
--- dmenu/dmenu.c	2015-10-02 15:50:07.703727443 +0200
+++ dmenu-modified/dmenu.c	2015-10-02 15:49:35.660197418 +0200
@@ -542,7 +542,7 @@ setup(void)
 	utf8 = XInternAtom(dpy, "UTF8_STRING", False);
 
 	/* calculate menu geometry */
-	bh = drw->fonts[0]->h + 2;
+	bh = drw->fonts[0]->h + 12;
 	lines = MAX(lines, 0);
 	mh = (lines + 1) * bh;
 #ifdef XINERAMA
